db/migrate/20231129162642_create_posts.rb

```diff
@@ -0,0 +1,13 @@
+class CreatePosts < ActiveRecord::Migration[7.1]
+  def change
+    create_table :posts do |t|
+      t.references :author, null: false, foreign_key: true
+      t.string :title
+      t.text :text
+      t.integer :comments_counter
+      t.integer :likes_counter
+
+      t.timestamps
+    end
+  end
+end
```

Comment:
- [ ] When referencing the user table here we have to do so explicitly as if we dont this will cause errors as the line of code No. 4 assumes w have a table titled `author`. This will cause an error going forward. Instead use the following approach:

e.g
```
`t.references :author, foreign_key: { to_table: 'users' }`
```